src/components/accessories/patientNewBill/styles.scss

```diff
@@ -0,0 +1,118 @@
+@import "../../../styles/variables";
+@import "../../../../node_modules/susy/sass/susy";
+
+.patientNewBill {
+  display: flex;
+  flex-flow: column;
+  border: 1px solid hsl(2, 98%, 53%, .075);
+  border-radius: 8px;
+  box-shadow: 2px 2px 2px hsl(2, 98%, 53%, .075);
+  padding: 16px 8px 16px 8px;
+  row-gap: 1em;
+  & > div:first-child {
+    display: flex;
+    column-gap: 1em;
+    @include susy-media($smartphone) {
+      row-gap: 2em;
+      flex-flow: column; 
+      align-items: center; 
+    }
+    & > .patientNewBill_left {
+      display: flex;
+      flex: 9;
+      justify-content: space-between;
+      flex-flow: column;
+      row-gap: 1em;
+      background-color: white;
+      & > .patientNewBill_buttons {
+        display: flex;
+        column-gap: 2em;
+        justify-content: space-between;
+      }
+      .addpayment {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        color: gray;
+      }
+      @include susy-media($smartphone) {
+        width: 1fr;
+      }
+    }
+    & > .patientNewBill_right {
+      display: flex;
+      flex-flow: column;
+      align-items: center;
+      background-color: hsl(2, 98%, 53%, .3);
+      border-radius: 8px;
+      padding: 1.5em .25em 1.5em .25em;
+      row-gap: 1em;
+      .smallButton, .textButton {
+        width: 10em;
```

Comment:
It's not a best practice use "em" as unit of measure of height, width or position. It's better use px and % for this.